src/lib/core/styles/tokens/typography/_tokens.label.scss

```diff
@@ -10,6 +10,13 @@ $label: utils.inherit-map(core.$base, (
   letter-spacing: type-utils.calc-letter-spacing(0.5, scale.value('0875'))
 )) !default;
 
+// TODO: solidify label 2 styles
+$label2: utils.inherit-map(core.$base, (
+  font-size: type-utils.font-size-relative(label, font-size, '0813'),
+  line-height: type-utils.font-size-relative(label, line-height, '1250'),
```

Comment:
We may want to use a `normal` line height on this to ensure wrapping labels aren't too space apart. I'd also consider doing this on the standard "label" style as well?